helix-core/src/main/java/org/apache/helix/util/InstanceValidationUtil.java

```
@@ -464,15 +537,15 @@ public static boolean siblingNodesActiveReplicaCheck(HelixDataAccessor dataAcces
             }
           }
           if (numHealthySiblings < minActiveReplicas) {
-            _logger.info(
-                "Partition {} doesn't have enough active replicas in sibling nodes. NumHealthySiblings: {}, minActiveReplicas: {}",
-                partition, numHealthySiblings, minActiveReplicas);
-            return false;
+            _logger.warn(
```

I would suggest to move this to debug log instead. IDTS it qualifies as WARN. 
Also, debug because now we dont just check first failure, we keep going even after the faiilures, so this becomes a Hot path now. In the end we can post a summary.

---

> we keep going even after the faiilures, so this becomes a Hot path now.

Nope. we are still failing fast and provide only the details of first partition that's violating the min active replica check. And I chose warn as this may require attention and helpful while debugging.